script.js

```diff
@@ -1,4 +1,138 @@
+// Base version of Beat That
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+//define global variables
+var GAME_STATE_DICE_ROLL = "GAME_STATE_DICE_ROLL";
```

Comment:
Nit pick: I would remove the GAME_STATE words from the variable name and value.
e.g. var ROLLING_DICE = "ROLLING_DICE"
also, in the value of your variable, you don't need the underscores.